package.json

```diff
@@ -1,41 +1,73 @@
 {
   "name": "vis-util",
-  "version": "1.0.2",
+  "version": "1.1.0",
   "description": "utilitie collection for visjs",
   "main": "dist/umd.js",
   "module": "dist/esm.js",
-  "homepage": "https://github.com/visjs/vis-util",
-  "license": "(Apache-2.0 OR MIT)",
+  "types": "dist/types/index.d.ts",
   "repository": {
     "type": "git",
-    "url": "git://github.com/visjs/vis-utils.git"
-  },
-  "bugs": {
-    "url": "https://github.com/visjs/vis-util/issues"
+    "url": "git+https://github.com/visjs/vis-utils.git"
   },
   "author": "mojoaxel <dev@wunschik.net>",
-  "keywords": [],
+  "contributors": [
+    "Thomaash <tom.vycital@gmail.com>"
+  ],
   "scripts": {
-    "build": "npm run build:code",
+    "build": "npm run build:types && npm run build:code && npm run build:docs",
     "build:code": "rollup -c rollup.prod.config.js",
+    "build:docs": "typedoc ./src",
+    "build:types": "tsc -p tsconfig.types.json",
     "clean": "rimraf '{docs,dist}/*'",
-    "test": "mocha"
+    "fix": "npm run lint -- --fix",
+    "lint": "eslint '{src,test}/**/*.{m,}{j,t}s{x,}'",
+    "test": "BABEL_ENV=test mocha -r ./babel.mocha.js 'test/**/*.test.{j,t}s'",
+    "type-check": "tsc --noemit"
+  },
+  "engines": {
+    "node": ">=8"
   },
+  "keywords": [
+    "util",
+    "vis",
+    "vis.js"
+  ],
+  "license": "(Apache-2.0 OR MIT)",
+  "bugs": {
+    "url": "https://github.com/visjs/vis-util/issues"
+  },
+  "homepage": "https://github.com/visjs/vis-util",
+  "files": [
+    "dist",
+    "LICENSE*"
+  ],
   "dependencies": {
+    "moment": "^2.24.0",
```

Comment:
:+1:

Reply:
Thinking about this, shouldn't Moment be a peer dependency?

Reply:
You are write! I'm not sure how to handle this at the moment. For the browser version we definitely need to generate a bundle without moment.
Feel free to provide a solution for this. For now on I'm also ok with bundling moment.js by defrault and remove it in a later version...not ideal..

Reply:
I'll look into this.

Reply:
Here is how vis used to handle it: https://github.com/visjs/vis_original/blob/master/lib/module/moment.js

Reply:
But then it always has to be bundled, it only remains unused if something else already imported Moment.

Reply:
Peer dependency would be more along the lines “shipping this is your job, not ours”.

Reply:
Maybe multiple builds? With and without Moment, so that users can pick which better suits their needs?

Reply:
We can create builds without peer dependencies (smaller files for people who install through npm or ship their own) and with dependencies (bigger files for people who just want it to work) like this:
```javascript
import babel from 'rollup-plugin-babel'
import resolve from 'rollup-plugin-node-resolve'
import packageJSON from './package.json'

// This is necessary for Moment to work.
import commonjs from 'rollup-plugin-commonjs'

const external = Object.keys(packageJSON.peerDependencies || {})

const plugins = [
  resolve({
    extensions: ['.ts', '.js', '.json'],
  }),
  commonjs(),
  babel({
    extensions: ['.ts', '.js'],
  }),
]

export default [
  {
    input: 'src/index.ts',
    output: {
      file: 'dist/esm.js',
      format: 'esm',
    },
    external,
    plugins,
  },
  {
    input: 'src/index.ts',
    output: {
      file: 'dist/esm.with-deps.js',
      format: 'esm',
    },
    plugins,
  },
  {
    input: 'src/index.ts',
    output: {
      file: 'dist/umd.js',
      format: 'umd',
      name: 'visUtil',
    },
    external,
    plugins,
  },
  {
    input: 'src/index.ts',
    output: {
      file: 'dist/umd.with-deps.js',
      format: 'umd',
      name: 'visUtil',
    },
    plugins,
  },
]
```

Reply:
Look interesting.
This is something we can discuss further on the main packeges: vis-timeline, vis-network and vis-graph3d. 
For now we could leave it as it is!

Reply:
Okay.